templates/gbdb.conf.erb

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
@ehelms, updated so that script reads the gutterball conf
@ekohl couldn't do anything about the password since it seems required option by liquibase to be passed in. http://www.liquibase.org/documentation/command_line.html#required-parameters